e2e-tests/development-runtime/cypress/integration/functionality/query-data-caches.js

```diff
@@ -110,7 +110,9 @@ function pageTitleAndDataAssertion(config) {
 function runTests(config) {
   preTestSetup(config)
 
-  cy.visit(`/query-data-caches/${config.slug}/page-A/`).waitForRouteChange()
+  cy.visit(`/query-data-caches/${config.slug}/page-A/`, {
+    failOnStatusCode: config.initialPage !== `404`,
+  }).waitForRouteChange()
```

Comment:
`cy.visit` would fail if status is 404 - we do want to continue (as we do expect this code for 404 page)